lib/Page.js

```diff
@@ -36,34 +36,41 @@ class Page extends EventEmitter {
    * @return {!Promise<!Page>}
    */
   static async create(client, ignoreHTTPSErrors, appMode, screenshotTaskQueue) {
+
+    await client.send('Page.enable');
+    const {frameTree} = await client.send('Page.getResourceTree');
+    const page = new Page(client, frameTree, ignoreHTTPSErrors, screenshotTaskQueue);
+
     await Promise.all([
       client.send('Network.enable', {}),
-      client.send('Page.enable', {}),
       client.send('Runtime.enable', {}),
       client.send('Security.enable', {}),
     ]);
     if (ignoreHTTPSErrors)
       await client.send('Security.setOverrideCertificateErrors', {override: true});
-    const page = new Page(client, ignoreHTTPSErrors, screenshotTaskQueue);
-    await page.goto('about:blank');
     // Initialize default page size.
     if (!appMode)
       await page.setViewport({width: 800, height: 600});
+
+    // The page might still be initializing, if so wait for the first navigation
```

Comment:
@JoelEinbinder could you please check with @dgozman? this looks like something that should not be happenning

Reply:
"It is good enough" - @dgozman

Reply:
I don't understand why we do this here. Doesn't it make more sense in target reporting code then?

Reply:
It is accurately reporting whats going on in the browser. it just is inconvenient for users of `Page` so we wait for first navigation.

Reply:
is there any test that would help me understand what's going on?